presto-docs/src/main/sphinx/release/release-306.rst

```diff
@@ -0,0 +1,76 @@
+===========
+Release 306
+===========
+
+General Changes
+---------------
+
+* Fix planning failure for queries containing a ``LIMIT`` after a global
+  aggregation. (:issue:`437`)
+* Fix missing column types in EXPLAIN output. (:issue:`328`)
+* Fix updating peak revocable memory reservation. (:issue:`413`)
+* Fix blocking on memory when revocable memory is used. (:issue:`413`)
+* Fix double memory accounting for spilling aggregations. (:issue:`413`)
+* Fix possible CPU active waiting when window operator spill is triggered. (:issue:`468`)
+* Fix incorrect view name displayed by ``SHOW CREATE VIEW``. (:issue:`433`)
+* Allow specifying ``NOT NULL`` when creating tables or adding columns. (:issue:`418`)
+* Add a config option (``query.stage-count-warning-threshold``) to specify a
+  per-query threshold for the number of stages. When this threshold is exceeded,
+  a ``TOO_MANY_STAGES`` warning is raised. (:issue:`330`)
+* Support session properties with comma or equality sign. (:issue:`407`)
```

Comment:
> Support session property values with a comma or equals sign.

Or should we mention other special characters?

> Support session property values with special characters (e.g., comma or equals sign).

Reply:
The second one sounds good.